client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -342,32 +342,21 @@ class AdsFormSettings extends Component {
 				<FormFieldset>
 					<SupportInfo
 						text={ translate(
-							'Enables a targeted advertising opt-out link for California consumers, as required by the California Consumer Privacy Act (CCPA).'
+							'Enables a targeted advertising opt-out link for consumers from some US states.'
 						) }
-						link="https://wordpress.com/support/your-wordpress-com-site-and-the-ccpa/"
+						privacyLink={ null }
```

Comment:
Temporarily removed the link because the linked support page includes now outdated CCPA content that needs to be changed. This link would be updated once we have new support page for this.